internal/utility.go

```diff
@@ -182,3 +183,11 @@ func stripWalFileName(path string) string {
 	}
 	return ""
 }
+
+type ForbiddenActionError interface {
```

Comment:
Add here tracelog printing (function Error), like in other errors.

Reply:
done